### ISSUE TITLE
Doc: The description for `file` in `get_all_tweets` is not correct

### ISSUE THREAD

**Issue by chainsawriot** [cjbarrie/academictwitteR#121]
**Describe the bug**

According to [the documentation](https://github.com/cjbarrie/academictwitteR/blob/5c3a38c2b5fac9a74accb5b6da95818c69565b9a/R/get_all_tweetsv2.R#L5):

     If a filename is supplied, this function will save the result as a RDS file, **otherwise** it will return the results as a dataframe.  

Apologize that English is not my first language, but from my reading it shouldn't return a dataframe when a filename is supplied. But the code [says otherwise](https://github.com/cjbarrie/academictwitteR/blob/5c3a38c2b5fac9a74accb5b6da95818c69565b9a/R/utils.R#L128).

It also raises questions about the return value as described in the documentation. It is not always a data frame as the documentation said. It should only return a data frame when `bind_tweets` is `TRUE`.

**To Reproduce**
``` r
require(academictwitteR)
#> Loading required package: academictwitteR
require(testthat)
#> Loading required package: testthat
data_dir <- paste0(tempdir(), "/", paste0(sample(letters, 12), collapse = ""))

x <- expect_silent(get_all_tweets("#commtwitter", start_tweets = "2021-06-01T00:00:00Z", 
               end_tweets = "2021-06-05T00:00:00Z", bind_tweets = TRUE, 
               data_path = data_dir, verbose = FALSE, file = "whatever.RDS"))
expect_true(file.exists("whatever.RDS"))
expect_false(is.data.frame(x))
#> Error: is.data.frame(x) is not FALSE
#> 
#> `actual`:   TRUE 
#> `expected`: FALSE
unlink(data_dir, recursive = TRUE)
unlink("whatever.RDS")
```

<sup>Created on 2021-06-10 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Expected behavior**

Either 1) not returning a data frame as the documentation described, or 2) clarify what is actually going on in the documentation.

**Comment by cjbarrie** [comment]
You're right: the language is unclear. The function will return a .rds file *and* a data frame when `bind_tweets = T`. The language needs to be cleaned up. I can add this to Projects as task that I am happy to complete

**Comment by cjbarrie** [comment]
Corrected in #158
